crates/re_viewer/src/ui/rerun_menu.rs

```diff
@@ -385,9 +385,16 @@ fn debug_menu_options_ui(
     ).on_hover_text("Show a debug overlay that renders the picking layer information using the `debug_overlay.wgsl` shader.");
 
     re_ui.checkbox(ui,
-        &mut app_options.show_blueprint_in_timeline,
-        "Show Blueprint in the Time Panel",
-    ).on_hover_text("Show the Blueprint data in the Time Panel tree view. This is useful for debugging the internal blueprint state.");
+        &mut app_options.show_blueprint_timeline,
+        "Show a timeline of the blueprint data.",
+    )
+    .on_hover_text("Show a timeline of the blueprint data. This is useful for debugging the internal blueprint state.");
+
+    re_ui.checkbox(ui,
+        &mut app_options.disable_blueprint_gc,
+        "Disable the blueprint garbage collector.",
+    )
+    .on_hover_text("Don't garbage collect the blueprint data. This is useful for debugging the internal blueprint state.");
```

Comment:
I think this too would be be better as a radio pair, but for a different reason: to avoid a double negative (unchecked checkbox currently means "do not don't disable garbage collection")